### ISSUE TITLE
Feature request: resize SVG on import

### ISSUE THREAD

**Issue by fsih** [LLK/scratch-paint#578]
Resize svg on import feature??? @BryceLTaylor

**Comment by fsih** [comment]
I believe the context was he imported an SVG that was much larger than the editor window and it was very difficult to resize down

**Comment by apple502j** [comment]
Support: In Scratch 2.0 too. I've been using Pixabay's svg files, but it's too large to use on Scratch, so I need to resize on Inkscape.

**Comment by BryceLTaylor** [comment]
Yeah, that was the gist of it.  If an SVG is large it extends beyond the editor and it's very hard to scale it down or even find a smaller part that you wanted.  I pulled in one from the internet, I can't find the particular image at the moment, and it was massive.

I think in general it would make sense to default to fitting it on the screen and then the user could scale it up if that's what they wanted.  It's not always clear how big an SVG is before importing

**Comment by fsih** [comment]
For example, import https://s3-us-west-2.amazonaws.com/s.cdpn.io/183091/harper-coloring-book009.svg
![image](https://user-images.githubusercontent.com/2855464/44165286-4fd99c00-a096-11e8-849d-aa96842d4a09.png)

**Comment by fsih** [comment]
This will become more relevant when https://github.com/fsih/scratch-paint/tree/fillSpace is merged

**Comment by fsih** [comment]
The design should probably be similar to what oversized bitmaps do on import. I think one concern we might have had was if people want to download their costumes from Scratch 2 and upload them to Scratch 3 looking exactly the same, they might get downsized.
